manager/catalog_manager.go

```diff
@@ -66,13 +64,15 @@ func SetEnv() {
 //Init clones or pulls the catalog, starts background refresh thread
 func Init() {
 	_, err := os.Stat(catalogRoot)
-	if err == nil {
+	if os.IsExist(err) || err == nil {
```

Comment:
This needs to be if !os.IsNotExist(err) || err == nil {..}

os.IsNotExist(err) will be true if the dir does not exists. We will just negate it to check if dir exists.